util/command/download.py

```diff
@@ -39,7 +39,7 @@ def run(self):
                     # skip the top-level directory
                     if obj.key == selected_area:
                         continue
-                    fs.append(FileTransfer(key=obj.key, size=obj.size))
+                    fs.append(FileTransfer(path='.', key=obj.key, size=obj.size))
```

Comment:
just realised, might be better to use python method os.getcwd()

Reply:
totally right